packages/cadl-ranch-specs/http/parameters/spread/main.tsp

```diff
@@ -248,16 +289,55 @@ namespace Alias {
   Expected request body:
   ```json
   {
-    "prop1":"foo1",
-    "prop2":"foo2",
-    "prop3":"foo3",
-    "prop4":"foo4",
-    "prop5":"foo5",
-    "prop6":"foo6"
+    "requiredString": "foo",
+    "optionalInt": 1,
+    "requiredIntList": [1, 2],
+    "optionalStringList": ["foo", "bar"]
+
   }
   ```
   """)
   @route("/multiple-parameters/{id}")
   @put
   op spreadWithMultipleParameters(...MultipleRequestParameters): NoContentResponse;
+
+  alias ThingAlias = {
+    @doc("name of the Thing")
+    name: string;
+
+    @doc("age of the Thing")
+    age: int32;
+  };
+
+  alias ThingAliasWithSpreadAlias = {
```

Comment:
I think we need a more comprehensive name for this alias.

Reply:
This Alias ​​has been modified to InnerAliasParameter according to PanShao's comment.